policies/mocks/policies.go

```diff
@@ -32,6 +32,19 @@ func (m *mockPoliciesRepository) ActivateDatasetByID(ctx context.Context, datase
 	return nil
 }
 
+func (m *mockPoliciesRepository) RetrieveAllPoliciesInternal(ctx context.Context, ownerID string) ([]policies.Policy, error) {
+	var policyList []policies.Policy
+	id := uint64(0)
+	for _, p := range m.pdb {
+		if p.MFOwnerID == ownerID {
+			policyList = append(policyList, p)
+		}
+		id++
+	}
```

Comment:
here I don't understand why increment the `var id` if not append `policyList`, also the `var id` is not used for reading anymore, I think we can remove it.